entries/g/growingio.com.json

```diff
@@ -4,7 +4,7 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://docs.growingio.com/docs/product-manual/sysmanage/personal/security#kai-qi-er-ci-yan-zheng",
+    "documentation": "https://docs.growingio.com/v3/product-manual/personal/security",
```

Comment:
I'm getting a timeout error when I go to this link. Is it happening for anyone else?

Reply:
Maybe you're blacklisted by the CCP :laughing: 
Works for me